recipes/aws-c-event-stream/all/conanfile.py

```diff
@@ -73,12 +73,12 @@ def package(self):
 
     def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "aws-c-event-stream")
-        self.cpp_info.set_property("cmake_target_name", "AWS")
+        self.cpp_info.set_property("cmake_target_name", "AWS::AWS")
```

Comment:
```suggestion
        self.cpp_info.set_property("cmake_target_name", "AWS::aws-c-event-stream")
```

Like `vulkan-headers` recipe, very important to not define `AWS::AWS` target. It was a tedious issue of previous mechanism.